sql/hive-thriftserver/src/main/java/org/apache/hive/service/auth/HiveAuthFactory.java

```diff
@@ -285,9 +288,10 @@ public String verifyDelegationToken(String delegationToken) throws HiveSQLExcept
     try {
       return delegationTokenManager.verifyDelegationToken(delegationToken);
     } catch (IOException e) {
-      String msg =  "Error verifying delegation token " + delegationToken;
-      LOG.error(msg, e);
-      throw new HiveSQLException(msg, "08S01", e);
+      LOG.error("Error verifying delegation token {}", e,
+        MDC.of(LogKeys.TOKEN$.MODULE$, delegationToken));
+      throw new HiveSQLException("Error verifying delegation token " + delegationToken,
```

Comment:
nit: we can store the log entry into `msg` and use `msg.message` here.

Reply:
How about the following?
<img width="642" alt="image" src="https://github.com/apache/spark/assets/15246973/ed274dd6-27ca-48b1-ad47-e1cc364d5ccb">